### PR TITLE
CI/CD GH releases + rework

### PR DESCRIPTION
**Pull request by ysibirski**
## Description
Modifying CI/CD by simplifying + GH releases for DEV/PROD release build
This PR includes timeout issue for one of the tests as well

## Jira Ticket
https://horizenlabs.atlassian.net/browse/DEVOP-835

## Changes
GH Releases + rework of CI/CD

## Checks
- [ yes ] Project Builds
- [ yes ] Project passes tests and checks
 
## Additional information
I need to monitor it next time we will build a RELEASE

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,27 +15,25 @@ git:
...
 
 jobs:
   include:
-    - name: jdk-11_latest
-      env: PUBLISH_BUILD=true IMAGE_TAG='bionic_jdk-11_latest' TESTS='000'
-
+    - stage: java project
+      name: jdk-11_latest
+      script: ci/script.sh
+      env: DOCKER_ORG='zencash' IMAGE_NAME='sc-ci-base' IMAGE_TAG='bionic_jdk-11_latest' TESTS='000'
```
Comment (otoumas) [inline .travis.yml:30]: Maybe the list nomenclature could be used here:
```
- stage: java project
      name: jdk-11_latest
      script: ci/script.sh
      env: 
          - DOCKER_ORG='zencash' 
          - IMAGE_NAME='sc-ci-base' 
          - IMAGE_TAG='bionic_jdk-11_latest' 
          - TESTS='000'
```

File: ci/docker/entrypoint_setup_gpg.sh
Comment (otoumas) [inline ci/docker/entrypoint_setup_gpg.sh:1]: Where is this entrypoint used?

**Review by otoumas — APPROVED** [review]
LGTM, see coments

File: .travis.yml (hunk 1 above)
Comment (ysibirski) [reply .travis.yml:30]: It can be, just more lines, but probably easier to read?

File: ci/docker/entrypoint_setup_gpg.sh
Comment (ysibirski) [reply ci/docker/entrypoint_setup_gpg.sh:1]: This one is tricky :) it is being used under `entrypoint.sh` and if it is A_RELEASE then takes care of gpg config for signing the artifacts

```
if [ "$USERNAME" = "root" ]; then
  exec /build/ci/docker/entrypoint_setup_gpg.sh "$@"
else
  exec gosu "$USERNAME" /build/ci/docker/entrypoint_setup_gpg.sh "$@"
fi
```

File: .travis.yml (hunk 1 above)
Comment (cronicc) [inline .travis.yml:30]: TESTS_IMAGE_TAG should be switched to focal.

Comment (otoumas) [reply .travis.yml:30]: I usually prefer multiple lines, easier to read and to find envars

File: ci/docker/entrypoint_setup_gpg.sh
Comment (otoumas) [reply ci/docker/entrypoint_setup_gpg.sh:1]: It was tricky yes!

**Comment by ysibirski** [comment]
@cronicc and @otoumas  both of your comments were addressed and the build was tested successfully.

**Review by cronicc — APPROVED** [review]

**Pull request merged**